R/rhino.R

```diff
@@ -52,29 +52,31 @@ copy_rproj <- function() {
   )
 }
 
-system_cmd_version <- function(cmd) {
-  system2(cmd, "--version", stdout = TRUE, stderr = TRUE)
-}
-
-print_system_cmd_version <- function(cmd) {
+system_cmd_version <- function(cmd, throw_error = FALSE) {
   tryCatch(
-    system_cmd_version(cmd),
-    error = function(e) e$message
+    system2(cmd, "--version", stdout = TRUE, stderr = TRUE),
+    error = function(e) {
+      if (isTRUE(throw_error)) stop(e)
```

Comment:
Though right now we will always catch this, it's probably better to use `cli_abort()` for consistent printing if we refactor:
```suggestion
      if (isTRUE(throw_error)) cli::cli_abort(e)
```